cli/command/trust/info.go

```diff
@@ -119,23 +119,27 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 
 	// This will always have the root and targets information
-	fmt.Fprintf(cli.Out(), "\nList of admins and their KeyIDs:\n\n")
-	printSignerInfo(cli, adminRoleToKeyIDs)
+	fmt.Fprintf(cli.Out(), "\nRepository keys:\n\n")
```

Comment:
can we add the repo name here?  Also, how does this look without the double newline?